package.json

```diff
@@ -13,13 +13,14 @@
     "react-firestore": "^1.0.1",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.3.0"
+    "react-scripts": "3.3.0",
+    "semantic-ui-react": "^0.88.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "start": "craco start",
```

Comment:
Was this to allow overriding some of semantic ui's `less` styling?

Reply:
Yes, the Semantic UI docs recommended craco so that the app doesn't eject when the less for the custom theme compiles. I'm not too familiar with that but that's how I understood it.

https://react.semantic-ui.com/theming/